USAGE.md

```diff
@@ -327,6 +327,25 @@ Potential spelling errors will be printed in the continuous integration log alon
 Words in `build/assets/custom-dictionary.txt` are ignored during spellchecking.
 Spellchecking is currently only supported for English language manuscripts.
 
+## AI-assisted authoring
+
+The workflow [`ai-revision`](.github/workflows/ai-revision.yaml) is available to assist authors in writing their manuscripts.
+It uses large language models (LLM) to revise the manuscript text, fixing spelling and grammar errors, and improving the sentence structure and the writing style with section-specific prompts.
+It is manually triggered by the user (it never runs automatically), and it generates a pull request with suggested revisions.
+Then the user can review these changes and merge the pull request if they are acceptable.
+More information about this tool is available in [this manuscript](https://greenelab.github.io/manubot-gpt-manuscript/).
+
+You must provide a secret with name `OPENAI_API_KEY` containing your OpenAI API token.
+The easiest way to set this is to go to your organization's settings page and, within "Secrets and variables," select "Actions."
```

Comment:
```suggestion
The easiest way to set this is to go to your GitHub user or organization's settings page and, within "Secrets and variables," select "Actions."
```
This phrasing is awkward, but what is a better way to be inclusive or individual GitHub users as well?